README.md

```diff
@@ -1 +1 @@
-# data-feed-proxy-combinators
\ No newline at end of file
+# data-feed-proxy-combinators
```

Comment:
Yeah, we can add this later. It's not the ideal order, but will do.

Then I'll get you in touch with the auditors and let you handle this e2e.